pkg/kubelet/network/cni/cni.go

```diff
@@ -271,7 +272,11 @@ func (network *cniNetwork) deleteFromNetwork(podName string, podNamespace string
 	err = cninet.DelNetwork(netconf, rt)
 	if err != nil {
 		glog.Errorf("Error deleting network: %v", err)
-		return err
+
+		subStr := "no such file or directory"
```

Comment:
No. We can't. We are parsing the output of an external command on json format, with a message attribute (the one we are parsing) and a error code which is dropped.. There is a PR containernetworking/cni#406 to not drop the cni error code, so in the future we can have more control about it, but that will imply update the cni vendor to the latest version.